src/providers/ipa/ipa_opts.c

```diff
@@ -323,6 +323,7 @@ struct sdap_attr_map ipa_override_map[] = {
     { "ldap_group_gid_number", "gidNumber", SYSDB_GIDNUM, NULL },
     { "ldap_user_ssh_public_key", "ipaSshPubKey", SYSDB_SSH_PUBKEY, NULL },
     { "ldap_user_certificate", "userCertificate;binary", SYSDB_USER_CERT, NULL },
+    { "", "objectClass", SYSDB_OBJECTCLASS, NULL }, /* We don't want this to be configurable */
```

Comment:
Hi,

I think it would be better to use something like `SYSDB_ORIG_OBJECTCLASS` and define this as `originalObjectClass` or similar to avoid issue with idoverride objects which are written into the cache. For the 'Default Trust View' the idoverride data is written directly into the cached user object, but for other views there will be an object in the cache as well.

bye,
Sumit

Reply:
Done